documentation/examples/prometheus-scaleway.yml

```diff
@@ -0,0 +1,29 @@
+# A example scrape configuration for running Prometheus with Scaleway.
+scrape_configs:
+  - job_name: 'instance'
+    scaleway_sd_configs:
+      - role: instance
+        project: 11111111-1111-1111-1111-111111111111
```

Comment:
I guess most users would not need this, I would not keep this in the example.

Reply:
I would prefer to keep it. Projects are one of our building blocks for building up IAM features. So I would prefer users to keep track of the project their resources are living on.

Reply:
Ideally project would be a label, not a config option, I would actually expect this to be a list ? Why should we limit to one project?

Reply:
From an API perspective, you always query by specifying one project. So if you have multiple projects you want to scrape you would create multiple jobs. The same way that you create two different jobs for both instance and bare-metal. If we have several projects we are forced to do aggregation and handle the case where one project will work and another won't. In the first contribution, I guess we can stay on one project per scraping job.